pepper-apis/src/main/java/org/broadinstitute/ddp/util/LiquibaseUtil.java

```diff
@@ -104,18 +106,18 @@ private void runPepperAPIsGlobalMigrations() throws LiquibaseException {
      *
      * @throws LiquibaseException if something goes wrong
      */
-    private void runHousekeepingGlobalMigrations() throws LiquibaseException {
+    private void runHousekeepingGlobalMigrations() throws LiquibaseException, SQLException {
         runMigrations(HOUSEKEEPING_GLOBAL_MIGRATIONS);
     }
 
     /**
      * Runs the specific changelog file. When migration fails, this will attempt to rollback the changes, as applicable.
      */
-    private void runMigrations(String changelogFile) throws LiquibaseException {
+    private void runMigrations(String changelogFile) throws LiquibaseException, SQLException {
         Liquibase liquibase = null;
         String tag = null;
         try {
-            liquibase = new Liquibase(changelogFile, new FileSystemResourceAccessor(), new JdbcConnection(conn));
+            liquibase = new Liquibase(changelogFile, new ClassLoaderResourceAccessor(), new JdbcConnection(dataSource.getConnection()));
             logLocks(liquibase.listLocks());
```

Comment:
This is the change that allow reading liquibase config files from class loader as resources rather than files.
GAE has very limited access to local file system.
We were already including them in the jars we were building, so not a big deal. And makes deployment faster (i.e. less files to upload)